Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+                        attachTo: { element: 'a.dropdown-item[href*="Page"]', on: 'top' },
+                        buttons: [
+                            backButton,
+                        ],
+                        id: 'flow_parts_content_items_new_page',
+                        when: {
+                            show() {
+                                addShepherdQueryParams();
+                                setWalkthroughCookies(this.tour.options.id, 'flow_parts_page_title');
+                            },
+                        },
+                    },
+                    {
+                        title: 'Flow parts',
+                        text: 'You can give it a title, just like for a blog post or for an article.',
+                        attachTo: { element: '#TitlePart_Title', on: 'top' },
+                        buttons: [
+                            {
+                                action: function () {
+                                    goToRelativePage(
+                                        Shepherd.activeTour.options.id, 'flow_parts_content_items_new', 'Admin', 'Admin/Contents/ContentItems');
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'flow_parts_page_title',
+                        when: {
+                            show() {
+                                preventSubmit();
+                            },
+                        },
+                    },
+                    {
+                        title: 'Flow parts',
+                        text: 'You can give it an URL, but you can leave it empty to auto-generate it!',
+                        attachTo: { element: '#AutoroutePart_Path', on: 'top' },
+                        buttons: [
+                            backButton,
+                            nextButton,
+                        ],
+                        id: 'flow_parts_page_permalink',
+                        when: {
+                            show() {
+                                preventSubmit();
+                            },
+                        },
+                    },
+                    {
+                        title: 'Flow parts',
+                        text: 'The page has a part called <i>"Flow part"</i>, this allows you to add different' +
+                            ' widgets to your page. If you want a simple <i>"page"</i>, with only HTML content, it\'s' +
+                            ' better to create an article. However if you want something more complex, and perhaps' +
+                            ' you created your own widget and you want to add that, then it\'s better to use a page.' +
+                            ' Click on the <i>"+"</i> icon.',
```

Comment:
I can't click on the + icon.

Reply:
Only after clicking on next.

Reply:
So this part is a bit misunderstanding.

Reply:
Yes true. "Click on the <i>"+"</i> icon." should be in the next step.